open-metadata-resources/open-metadata-deployment/charts/vdc/templates/ingress.yaml

```diff
@@ -1,72 +1,69 @@
+{{ if .Values.service.ingress.enabled  }}
 
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-    labels:
-        app.kubernetes.io/name: {{ include "myapp.name" . }}
-        helm.sh/chart: {{ include "myapp.chart" . }}
-        app.kubernetes.io/instance: {{ .Release.Name }}
-        app.kubernetes.io/managed-by: {{ .Release.Service }}
-        app.kubernetes.io/component: ingress
-    annotations:
-        ingress.bluemix.net/tcp-ports: "serviceName={{ include "myapp.fullname" . }}-ranger-service ingressPort=6080 servicePort=6080;serviceName={{ include "myapp.fullname" . }}-atlas-service ingressPort=21000 servicePort=21000;serviceName={{ include "myapp.name" . }}-cp-kafka ingressPort=9092 servicePort=9092;serviceName={{ include "myapp.name" . }}-openldap ingressPort=389 servicePort=389;serviceName={{ include "myapp.fullname" . }}-egeria-service ingressPort=8080 servicePort=8080;serviceName={{ include "myapp.fullname" . }}-gaian-service ingressPort=6144 servicePort=6144;serviceName={{ include "myapp.fullname" . }}-igxproxy-service ingressPort=8090 servicePort=8080;serviceName={{ include "myapp.fullname" . }}-postgresql-service ingressPort=5432 servicePort=5432;serviceName={{ include "myapp.fullname" . }}-ui-service ingressPort=8443 servicePort=8443"
+  labels:
+    app.kubernetes.io/name: {{ include "myapp.name" . }}
+    helm.sh/chart: {{ include "myapp.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: ingress
+  annotations:
+    ingress.bluemix.net/tcp-ports: "serviceName={{ include "myapp.fullname" . }}-ranger-service ingressPort=6080 servicePort=6080;serviceName={{ include "myapp.fullname" . }}-atlas-service ingressPort=21000 servicePort=21000;serviceName={{ include "myapp.name" . }}-cp-kafka ingressPort=9092 servicePort=9092;serviceName={{ include "myapp.name" . }}-openldap ingressPort=389 servicePort=389;serviceName={{ include "myapp.fullname" . }}-egeria-service ingressPort=8080 servicePort=8080;serviceName={{ include "myapp.fullname" . }}-gaian-service ingressPort=6144 servicePort=6144;serviceName={{ include "myapp.fullname" . }}-igxproxy-service ingressPort=8090 servicePort=8080;serviceName={{ include "myapp.fullname" . }}-postgresql-service ingressPort=5432 servicePort=5432;serviceName={{ include "myapp.fullname" . }}-ui-service ingressPort=8443 servicePort=8443"
 
-        #ingress.bluemix.net/rewrite-path: |
-        #  serviceName={{ include "myapp.fullname" . }}-ranger-service rewrite=/;
-        #  serviceName={{ include "myapp.fullname" . }}-atlas-service rewrite=/;
+    #ingress.bluemix.net/rewrite-path: |
+    #  serviceName={{ include "myapp.fullname" . }}-ranger-service rewrite=/;
+    #  serviceName={{ include "myapp.fullname" . }}-atlas-service rewrite=/;
 
-    name: {{ include "myapp.fullname" . }}-ingress
+  name: {{ include "myapp.fullname" . }}-ingress
```

Comment:
I wasn't able to use ingress satisfactorily on IBM cloud so intend to return to it.. so can't comment on whether this works or how it works across platforms

Reply:
I've only changed indentation to 2-space to match all the other YAMLs...